elixir/apps/web/lib/web/live/sign_in/email.ex

```diff
@@ -78,8 +77,8 @@ defmodule Web.SignIn.Email do
                     "block p-2.5",
                     "text-sm text-white font-medium",
                     "items-center text-center",
-                    "bg-primary-700 rounded-r",
-                    "focus:ring-4 focus:ring-primary-200 hover:bg-primary-800"
+                    "bg-accent-600 rounded-r",
+                    "focus:ring-4 focus:ring-accent-200 hover:bg-accent-700"
```

Comment:
Don't feel super strongly about this but can probably remove the focus states

```suggestion
                    "hover:bg-accent-700"
```

Reply:
I am happy to do that if you want, but I think if I remove the `focus` styles from things, it will just go to the default of putting a blue `focus` ring around things.  If that was what you had intended, happy to do it, but just wanted to make sure.

Reply:
Ah, that's a good catch. TIL about focus ring colors. Yeah, let's remove it and stick with the default for consistency. Later on we can get fancier and make the focus ring color a brand color for all clickable elements on product + website.

Reply:
Ok, sounds good 👍